src/cdk/scrolling/fixed-size-virtual-scroll.ts

```diff
@@ -112,34 +118,32 @@ export class FixedSizeVirtualScrollStrategy implements VirtualScrollStrategy {
     }
 
     const scrollOffset = this._viewport.measureScrollOffset();
-    const firstVisibleIndex = Math.floor(scrollOffset / this._itemSize);
-    const firstItemRemainder = scrollOffset % this._itemSize;
-    const range = this._expandRange(
-        {start: firstVisibleIndex, end: firstVisibleIndex},
-        this._bufferSize,
-        Math.ceil((this._viewport.getViewportSize() + firstItemRemainder) / this._itemSize) +
-            this._bufferSize);
-    this._viewport.setRenderedRange(range);
-    this._viewport.setRenderedContentOffset(this._itemSize * range.start);
-
-    this._scrolledIndexChange.next(firstVisibleIndex);
-  }
-
-  /**
-   * Expand the given range by the given amount in either direction.
-   * @param range The range to expand
-   * @param expandStart The number of items to expand the start of the range by.
-   * @param expandEnd The number of items to expand the end of the range by.
-   * @return The expanded range.
-   */
-  private _expandRange(range: ListRange, expandStart: number, expandEnd: number): ListRange {
-    if (!this._viewport) {
-      return {...range};
+    const firstVisibleIndex = scrollOffset / this._itemSize;
+    const range = {...this._viewport.getRenderedRange()};
```

Comment:
The spread here transpiles to a `for...in` loop with some logic in it. Since this method is being while scrolling, it might a good idea if we just constructed the range here ourselves, also consider that it's only two properties.